src/app/lib/config/services/ukrainian.js

```diff
@@ -138,6 +138,7 @@ export const service = {
       },
       media: {
         noJs: 'Ваш пристрій не підтримує відтворення мультимедійних файлів',
+        contentExpired: 'Контент більше не доступний.',
```

Comment:
Does this require a full stop at the end (none of the others do)

Reply:
It has the full stop in the spreadsheet. It probably doesn’t need it but can ask why it does.